contributing/documentation/standards.rst

```diff
@@ -137,8 +144,13 @@ Files and Directories
 English Language Standards
 --------------------------
 
-* **English Dialect**: use the United States English dialect, commonly called
-  `American English`_.
+Symfony documentation uses the United States English dialect, commonly called
+`American English`_. Disputes over vocabulary will always be resolved using the
+`American English Oxford Dictionary`_ reference. Disputes over neologisms not
+included yet in the dictionary will be resolved by Symfony Documentation maintainers.
```

Comment:
If you don't agree on this, I'll remove everything. The original idea was to provide a "protocol" to follow in case someone raises questions about uncommon words (refactorization, performant, front-end, etc.)  I believe in providing clear rules and enforce them equally to everybody. But again, if you don't agree on this, let's remove it.